burotel/indico_burotel/controllers.py

```diff
@@ -40,3 +41,10 @@ def _process_GET(self):
     def _process_POST(self, value):
         from indico_burotel.plugin import BurotelPlugin
         BurotelPlugin.user_settings.set(session.user, 'default_experiment', value)
+
+
+class RHRoomLongTermAttribute(RHProtected):
+    def _process(self):
+        long_term_rooms = {r.id: r.get_attribute_value('long-term')
```

Comment:
please write a query that gets it for all rooms. otherwise you are spamming queries